app/views/error_states/error_template.scala.html

```diff
@@ -29,7 +29,11 @@
     details: String*
 )(implicit request:RequestHeader, messages: Messages, appConfig: config.AppConfig)
 
-@layout(pageTitle = Some(pageTitle), helpAndSupport = false) {
+@layout(
+    pageTitle = Some(pageTitle),
+    helpAndSupport = false,
+    backLink = Some(routes.AuthorizedToViewController.onPageLoad().url)
```

Comment:
Should this specific backlink be hardcoded in the error template? I feel like it should be passed as a parameter.

Reply:
Passed as a parameter to the error template I mean.